src/libNode/ViewChangeBlockProcessing.cpp

```diff
@@ -238,19 +210,29 @@ bool Node::ProcessVCBlockCore(const VCBlock& vcblock) {
     return false;
   }
 
```

Comment:
I think you may need the mutex for DS comm. Also there is deperecared UpdateDSCommiteeCompositionAfterVC(deque ..) I see.

Reply:
implemented